### PR TITLE
Adding support for update test item API call

### PR DESCRIPTION
**Pull request by bigbZik**
Adding support for API call: `PUT /{projectName}/item/{item}/update`
Model: 
```
UpdateTestItemRQ {
description (string, optional),
tags (Array[string], optional)
}
```

### CODE REVIEW

File: reportportal_client/service.py [hunk 1]
```diff
@@ -213,6 +213,23 @@ def start_test_item(self, name, start_time, item_type, description=None,
         logger.debug("start_test_item - Stack: %s", self.stack)
         return item_id
 
+    def update_test_item(self, description=None, tags=None):
+        """Updates test item
```
Comment (iivanou) [inline reportportal_client/service.py]: This docstring does not correspond to PEP257.
D400, First line should end with a period. 
D401, First line should be in imperative mood.

File: reportportal_client/service_async.py [hunk 2]
```diff
@@ -284,6 +285,20 @@ def start_test_item(self, name, start_time, item_type, description=None,
         }
         self.queue.put_nowait(("start_test_item", args))
 
+    def update_test_item(self, description=None, tags=None):
+        """Updates test item
```
Comment (iivanou) [inline reportportal_client/service_async.py]: This docstring does not correspond to PEP257.
D400, First line should end with a period. 
D401, First line should be in imperative mood.

File: reportportal_client/service.py (hunk 1 above)
Comment (bigbZik) [reply reportportal_client/service.py]: Fixed

File: reportportal_client/service_async.py (hunk 2 above)
Comment (bigbZik) [reply reportportal_client/service_async.py]: Fixed

**Pull request merged**